Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -157,6 +163,18 @@ - (BOOL)rowAtIndexPathIsOnlyRowVisibleInSection:(NSIndexPath*)indexPath {
     }
 }
 
+/**
+ *  Check whether or not a section is going in or out of view.
+ *
+ *  @param indexPath The index path of the row which will or did end displaying.
+ *
+ *  @return @c YES if that section isn't displaying or the given row is the only one visible in its section, otherwise @c NO.
+ */
+- (BOOL)isVisibilityTransitioningForRowIndexPath:(NSIndexPath*)indexPath {
+    return ![self isDisplayingCellsForSection:indexPath.section]
```

Comment:
@bgestle true but the name is a lie. 

This is only if the method is called within the scope of didEndDispalyingCell. Called in any other context it may or may not be true.